### PR TITLE
new method: fetchConvertCurrencies

### PR DESCRIPTION
**Pull request by Dan-krm**
Added a new method to bitget, binance, okx and woo. I wasn't able to test on binance and woo because the endpoints aren't supported through their testnet.

### Bitget:
```
bitget.fetchConvertCurrencies ()
2024-04-04T08:31:49.903Z iteration 0 passed in 286 ms

{
  BTC: {
    info: {
      coin: 'BTC',
      available: '0.00009850',
      maxAmount: '0.754068',
      minAmount: '0.00001'
    },
    id: 'BTC',
    code: 'BTC',
    available: 0.0000985,
    minAmount: 0.00001,
    maxAmount: 0.754068
  },
}
```

### Binance:
```
[NotSupported] binance does not have a testnet/sandbox URL for sapi endpoints
```

### Okx:
```
okx.fetchConvertCurrencies ()
2024-04-04T09:30:33.341Z iteration 0 passed in 435 ms

{
  BTC: { info: { ccy: 'BTC', max: '', min: '' }, id: 'BTC', code: 'BTC' },
}
```

### Woo:
```
[BadRequest] woo {"success":false,"code":"-1006","message":"An internal error has occurred. We are unable to process your request. Please try again later."}
```

### CODE REVIEW

File: ts/src/base/Exchange.ts [hunk 1]
```diff
@@ -530,6 +530,7 @@ export default class Exchange {
                 'fetchClosedOrder': undefined,
                 'fetchClosedOrders': undefined,
                 'fetchClosedOrdersWs': undefined,
+                'fetchConvertCurrencies': undefined,
```
Comment (carlosmiei) [inline ts/src/base/Exchange.ts:533]: @Dan-krm two things,
First, can we add a proper return type?
Second, we need to add the stub implementation in the base so the c# recognizes it as an override

Comment (Dan-krm) [reply ts/src/base/Exchange.ts:533]: I'll add the stub and I think we can actually just use the Currencies return type

Comment (carlosmiei) [reply ts/src/base/Exchange.ts:533]: @Dan-krm yeah yeah we can use currencies, just saying that we should not add new untyped methods

Comment (Dan-krm) [reply ts/src/base/Exchange.ts:533]: I'll keep that in mind to make sure the new methods have a type

File: ts/src/binance.ts [hunk 2]
```diff
@@ -12415,4 +12416,66 @@ export default class binance extends Exchange {
...
+         * @method
+         * @name okx#fetchConvertCurrencies
+         * @description fetches all available currencies that can be converted
+         * @see https://www.okx.com/docs-v5/en/#funding-account-rest-api-get-convert-currencies
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} an associative dictionary of currencies
+         */
+        await this.loadMarkets ();
+        const response = await this.privateGetAssetConvertCurrencies (params);
+        //
```
Comment (carlosmiei) [inline ts/src/binance.ts:12431]: @Dan-krm this is okx implementation I think (inside binance)

Comment (Dan-krm) [reply ts/src/binance.ts:12431]: I'm not sure how I overlooked that, I'll update the binance implementation

**Pull request merged**